x/authz/client/testutil/cli_test.go

```diff
@@ -15,3 +15,7 @@ func TestIntegrationTestSuite(t *testing.T) {
 	cfg.NumValidators = 1
 	suite.Run(t, NewIntegrationTestSuite(cfg))
 }
+
+func TestGRPCQueryTestSuite(t *testing.T) {
+	suite.Run(t, new(GRPCQueryTestSuite))
```

Comment:
I think we should combine these 2 suites into 1. i.e. remove GRPCQueryTestSuite, and put its methods inside IntegrationTestSuite, maybe in a separate `grpc.go` file.

That way other chain developers only need to import 1 test suite.

Reply:
Because of the same `TestSuite`, few modules gRPC query tests failing.
We have 2 choices:
  1. Using a separate network for CLI and gRPC (Easy one).
  2. Fixing gRPC queries tests.

Reply:
I don't think it makes much sense to have two separate networks for different client entry points. (2) might be more burdensome, but I think it's the cleaner approach.